### PR TITLE
Removes relative pathing in hardhat.dm

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
I was investigating cakehat code and I saw this and it annoyed me

### CODE REVIEW

File: code/modules/clothing/head/hardhat.dm [hunk 1]
```diff
@@ -11,29 +11,27 @@
...
+		return
+	on = !on
+	icon_state = "hardhat[on]_[item_color]"
+	item_state = "hardhat[on]_[item_color]"
+	user.update_inv_head()	//so our mob-overlays update
 
-		if(on)	user.AddLuminosity(brightness_on)
-		else	user.AddLuminosity(-brightness_on)
+	if(on)	user.AddLuminosity(brightness_on)
+	else	user.AddLuminosity(-brightness_on)
```
Comment (duncathan) [inline code/modules/clothing/head/hardhat.dm]: Change these while you're at it; put the code on s separate line

**Comment by phil235** [comment]
:+1:

**Comment by duncathan** [comment]
I'm merging this early so that @swankcookie doesn't have to deal with conflicts

**Comment by swankcookie** [comment]
@duncathan thanks

**Pull request merged**